src/components/global/Button.jsx

```diff
@@ -1,16 +1,44 @@
 import React from "react";
 import { Link } from "react-router-dom";
+
 import getIcon from "./componentFunctions";
 
 function Button(props) {
-  const {path, text, icon, height, fontSize} = props;
+  const { path, text, icon, height, fontSize, handleOnClick } = props;
+
+  // Call click handler if user presses "enter" key on button
+  // Javascript keycode for enter key = 13
+  const handleKeyDown = (e) => {
+    if (e.keyCode === 13 && handleOnClick) {
```

Comment:
@Angstylemon Sorry for being picky - could we use `event.key === 'Enter'` to avoid magic number 13 in the future?

Reply:
I'll fix this up before it gets put into main, thanks for pointing it out!